openapi.yaml

```diff
@@ -3395,8 +3482,11 @@ components:
                     type: string
                 uri:
                     type: string
+                url:
+                    type: string
             example:
                 type: "original"
+                uri: "https://static.companycam.com/VuS0rPvrnNA1490797262.jpg"
                 url: "https://static.companycam.com/VuS0rPvrnNA1490797262.jpg"
```

Comment:
Looks like this line may have been duplicated

Reply:
Naah, this one is a URI vs the original URL. We send both in our responses and someone pointed out that we did not include both in the documnentation.